src/libraries/arrays/AddressArrays.sol

```diff
@@ -0,0 +1,29 @@
+pragma solidity ^0.4.13;
+
+
+library AddressArrays {
+    /*
+     * @dev Returns a slice of the array
+     * @param self The array to make a slice from.
+     * @param start The index to start at.
+     * @param size The target number of elements to return in the slice
+     * @return A newly allocated slice containing the requested elements
+     */
+    function slice(address[] _self, uint256 _start, uint256 _size) internal returns (address[]) {
```

Comment:
I added some tests for the uint256 variant. Parameterizing them for the address variant wouldn't be that hard if you think it's worth it.